ast/src/object/mod.rs

```diff
@@ -1,17 +1,17 @@
 use std::collections::BTreeMap;
 
-use crate::parsed::PilStatement;
+use crate::parsed::{asm::Params, PilStatement};
 
 mod display;
 
-#[derive(Default, Clone, PartialEq, Eq, PartialOrd, Ord)]
+#[derive(Default, Clone, PartialEq, Eq, PartialOrd, Ord, Debug)]
 pub struct Location {
-    path: Vec<String>,
+    limbs: Vec<String>,
```

Comment:
This got reverted back to `limbs` from `path`. I guess it needs a rebase on `main`.